src/spec/cli/cli.spec.ts

```diff
@@ -44,4 +68,16 @@ export class CliTests {
         // Assert
         expect.toBeTrue(command instanceof InitCommand);
     }
+
+    @Test('invalid reporter, should default to standard')
+    private async invalidReporter() {
+        // Act
+        const args = ['node', '/some/path', '-r', 'invalidReporter'];
+        const command = await this.cli.getCommand(args);
+
+        // Assert
+        expect.toBeTrue(command instanceof RunCommand);
+        const runCommand = command as RunCommand;
+        expect.toBeEqual(runCommand.testyConfigFile, 'testy.json');
```

Comment:
Not satisfied with this test as it only makes sure the CLI doesn't crash and doesn't really test the reporter.